onnxruntime/test/optimizer/graph_transform_test_builder.h

```diff
@@ -195,21 +196,20 @@ class ModelTestBuilder {
     return &graph_.GetOrCreateNodeArg(name, &type_proto);
   }
 
+  /// <summary>
+  /// Makes an initializer from the provided shape, element type, and raw data bytes.
+  /// </summary>
+  /// <param name="shape">Initializer shape</param>
+  /// <param name="elem_type">ONNX tensor element data type</param>
+  /// <param name="raw_data">Raw data bytes</param>
+  /// <returns>NodeArg pointer</returns>
+  NodeArg* MakeInitializer(gsl::span<const int64_t> shape, ONNX_NAMESPACE::TensorProto_DataType elem_type,
```

Comment:
I tried doing this, but this change would require modifying a large number of tests across various files. The main issue is that passing an initializer list, such as `{1, 2, 3}`, as an argument for a `gsl::span` parameter fails to compile. Would need to modify all call sites to pass in `AsSpan(std::array{1, 2, 3})` instead.

Although I think this would be a good cleanup task, I would prefer to keep this PR more focused.